api/openapi/v1/core-metadata.yaml

```diff
@@ -2235,18 +2054,41 @@ paths:
           type: string
       responses:
         200:
-          description: List of provision watchers associated to the device service, if any.
+          description: List of provision watchers associated with the device service
           content:
             '*/*':
               schema:
                 $ref: '#/components/schemas/provisionwatcher'
         400:
           description: For malformed or unparsable requests
         404:
-          description: If no provision watcher is associated with the provided device service, or if the device
-            service or provision watcher cannot be found.
+          description: If no device service match on the ID provided.
         500:
           description: For unknown or unanticipated issues.
+  /v1/provisionwatcher/{id}:
+    get:
```

Comment:
This is now duplicated

Reply:
- Cleaned up two relevant end points, and co-located them as well: (1) /v1/provisionwatcher/{id}, which is aligned with a GET, and (2) /v1/provisionwatcher/id/{id}, which is aligned with a DELETE 👍